packages/menu/src/FocusableMenuItem.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export interface FocusableMenuItemProps {
+  children: React.ReactElement;
+}
+
+const FocusableMenuItem = React.forwardRef(
```

Comment:
Can we add some inline docs on what this component's purpose is?

Reply:
👍